website/pages/docs/reference/manifest.mdx

```diff
@@ -159,6 +159,22 @@ Keys are human-readable link names, and values are URLs.
 "We're hiring" = "https://swmansion.com/careers/"
 ```
 
+## `[tool]`
+
+Scarb by default will warn about unused keys in Scarb.toml to assist in detecting typos and such. The `[tool]{:toml}` table,
+however, is completely ignored by Scarb and will not be warned about. This section can be used for tools which would like
+to store package configuration in Scarb.toml. For example:
```

Comment:
Please rewrite this sentence to first put an information what is this field about, and then later describe that Scarb will not warn here.